### DISCUSSION TITLE
.NET 6

### DISCUSSION THREAD

**Discussion by FoxTes** [PrismLibrary/Prism#2646] [General]
Hello. Is there any support for .NET 6 planned?

**Comment by FoxTes** [comment]
Sorry. Didn't see it discussed earlier.

Then I would like to know when the next release will be?

**Reply by dansiegel** [reply]
You do not need a binary built against .net6 for it to be supported with the exception of Maui. We will release when we feel there is a release warranted.

**Reply by FoxTes** [reply]
> Вам не нужен двоичный файл, созданный для .net6, чтобы он поддерживался, за исключением Maui. Мы выпустим, когда посчитаем, что выпуск оправдан.

Thanks for answering.
